hooks/useEthTransfer.ts

```diff
@@ -0,0 +1,33 @@
+import { Contract, utils } from "ethers"
+import { useEthersSigner } from "./useEthersSigner"
+import handleTxError from "../lib/handleTxError"
+import erc6551AccountAbi from "../lib/abi-ERC6551-account.json"
+
+const useEthTransfer = () => {
+  const signer = useEthersSigner()
+
+  const transferEthFromERC6551Account = async (
+    tokenBoundAccount: string,
+    to: string,
+    value: number,
+  ) => {
+    if (!signer) return { err: "missing signer" }
+    try {
+      const contract = new Contract(tokenBoundAccount, erc6551AccountAbi, signer)
+
+      const tx = await contract.executeCall(to, utils.parseEther(value.toString()), "0x")
```

Comment:
this isn't correct.

look at `transferERC721FromERC6551Account` for an example in existing codebase 

we can update in next PR